src/resources/js/Pages/Dashboard.vue

```diff
@@ -4,8 +4,13 @@
         <div class="container mx-auto">
             <!-- Page Heading -->
             <header class="bg-white shadow">
-                <div class="py-6 px-4 sm:px-6 lg:px-8">
-                    <h1 class="text-2xl">VaccinateOH Stats</h1>
+                <div class="flex items-center justify-between">
+                    <div class="py-6 px-4 sm:px-6 lg:px-8">
+                        <h1 class="text-2xl">VaccinateOH Stats</h1>
+                    </div>
+                    <div>
+                        <h1><a href=/>Take me to the map!</a></h1>
```

Comment:
I would avoid an H1 for this. Just using the link should be fine. With TailwindCSS, pretty much all styles are stripped, so using an h1 doesn't have any effect visually. And some quotes around the / would be good. :-)